distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Connection.enso

```diff
@@ -19,8 +19,8 @@ import project.Data.Dialect.Dialect
 import project.Data.SQL_Query.SQL_Query
 import project.Data.SQL_Statement.SQL_Statement
 import project.Data.SQL_Type.SQL_Type
-import project.Data.Table as Database_Table_Module
-import project.Data.Table.Table as Database_Table
+import project.Data.DB_Table as Database_Table_Module
+import project.Data.DB_Table.DB_Table as Database_Table
```

Comment:
I think that we could avoid the renaming where possible, so IMO this could be changed to:
```suggestion
import project.Data.DB_Table as DB_Table_Module
import project.Data.DB_Table.DB_Table
```